src/helper/html.rs

```diff
@@ -186,176 +180,98 @@ pub struct HfdElement {
     classes: Vec<String>,
 }
 impl HfdElement {
-    pub fn get_by_name(&self, name: &str) -> bool {
-        let mut result = false;
-        if self.name == name {
-            result = true;
-        }
-        result
+    pub fn has_name(&self, name: &str) -> bool {
+        self.name == name
     }
 
     pub fn get_by_name_and_attribute(&self, name: &str, attribute: &str) -> Option<String> {
-        let mut result: Option<String> = None;
-        if self.name == name {
-            match self.attributes.get(attribute) {
-                Some(v) => {
-                    result = Some(v.to_string());
-                }
-                _ => {}
-            }
-        }
-        result
+        self.attributes
+            .get(attribute)
+            .and_then(|v| (self.name == name).then(|| v.to_string()))
     }
 
-    pub fn get_by_class(&self, class: &str) -> bool {
-        let mut result = false;
-        for cls in &self.classes {
-            if class == cls {
-                result = true;
-            }
-        }
-        result
+    pub fn contains_class(&self, class: &str) -> bool {
```

Comment:
named as such because the function earlier name does not convey what it does. This is a **breaking** change.